src/utils/scrolling.ts

```diff
@@ -86,12 +86,10 @@ export function isScrollIntoViewCenteredAvailable(): boolean {
  * @return indicates if it succeeded
  */
 export function scrollIntoViewCenteredIfPossible(targetEl: HTMLElement): boolean {
-  if (!isScrollIntoViewCenteredAvailable()) {
```

Comment:
You can get rid of it. Integrator who are using it will stay on old version and with update then need to adjust.

Reply:
I had exported `isScrollIntoViewCenteredAvailable` only to use it in a test, which tested that the fallback works for older browsers: https://github.com/acrolinx/sidebar-sdk-js/blob/26c9cf84befe20f285dcc7ac06e25b1bfb779602/test/spec/adapters.ts#L801. 
In this case it scrolls to a different position, so the test must assert a different position if it runs in an older browser.
This said:
 * We could move isScrollIntoViewCenteredAvailable into the test or into test-utils.ts
 * We could use scrollIntoViewCenteredIfPossible with a dummy element as replacement for isScrollIntoViewCenteredAvailable in the test.
 * We could just remove the corresponding assertion in the test, assuming that nobody will run it ever again in an old browser.